packages/core/src/Queue.js

```diff
@@ -4,58 +4,60 @@
 
 // Based on https://github.com/petkaantonov/deque
 
-export default function Queue (capPow2) {
-  this._capacity = capPow2 || 32
-  this._length = 0
-  this._head = 0
-}
+export default class Queue {
+  constructor(capPow2) {
+    this._capacity = capPow2 || 32
+    this._length = 0
+    this._head = 0
+  }
 
-Queue.prototype.push = function (x) {
-  var len = this._length
-  this._checkCapacity(len + 1)
+  push(x) {
+    const len = this._length;
```

Comment:
We've been infiltrated by semicolons 😱 (a few below, too)

Reply:
semi-🚚 FTW 😆